build.xml

```diff
@@ -1,69 +1,69 @@
- 	<property name="wsdl.dir" value="wsdl"/>	
-	<property name="webinf.dir" value="tomcat/webapps/ROOT/WEB-INF"/>
-	<property name="app.lib.dir" value="${webinf.dir}/lib"/>
- 
-    <!--+
-        | Default classpath
-        +-->
-    <path id="classpath">
-        <fileset dir="${lib.dir}">
-        	<include name="**/*.jar" />
-        </fileset>
-    	<fileset dir="${app.lib.dir}">
-			<include name="**/*.jar" />
-		</fileset>
-        <pathelement path="${java.class.path}/" />
-    </path>
-    
-    <!--+
-        | Compiles the Java source files
-        +-->
-    <target name="compile" depends="clean">
-        <echo>Compiling classes</echo>
-        <javac destdir="${classes.dir}" classpathref="classpath" source="1.6" target="1.6" debug="on" >
-            <src path="src" />
-        </javac>
-    </target>
-	
-	<target name="dist" depends="compile">
-		<war destfile="${dist.dir}/ROOT.war" webxml="${webinf.dir}/web.xml">
-			<classes dir="${classes.dir}"/>
-			<lib dir="${app.lib.dir}"/>
-			<fileset dir="${root.dir}">
-				<exclude name="WEB-INF"/>
-			</fileset>
-		</war>
-	</target>
-    
-    <target name="generate-stubs" depends="clean">
-		<taskdef name="wsdl2java" classname="org.apache.axis.tools.ant.wsdl.Wsdl2javaAntTask" classpathref="classpath" />
-		<mkdir dir="${src.dir}"/>
-		<wsdl2java
-			url="${wsdl.dir}/asset-operation.wsdl"
-			debug="false"
-			all="true"
-			output="${src.dir}"
-		/>
-	</target>
-    
-    <target name="clean">
-        <echo>Cleaning the project</echo>
-        <delete dir="${classes.com.dir}" />
-        <delete dir="${work.dir}" />
-        <delete dir="${wtpwebapps.dir}" />
-    </target>
-    
-    
-</project>
+<project name="UMT">
+
+    <description>Universal Migration Tool</description>
+
+    <property name="src.dir" value="src"/>
+	<property name="dist.dir" value="dist"/>
+ 	<property name="lib.dir" value="tomcat/lib"/>
+ 	<property name="work.dir" value="tomcat/work"/>
+ 	<property name="wtpwebapps.dir" value="tomcat/wtpwebapps"/>
+	<property name="root.dir" value="tomcat/webapps/ROOT"/>
+ 	<property name="classes.dir" value="tomcat/webapps/ROOT/WEB-INF/classes"/>
+ 	<property name="classes.com.dir" value="${classes.dir}/com"/>
+ 	<property name="wsdl.dir" value="wsdl"/>	
+	<property name="webinf.dir" value="tomcat/webapps/ROOT/WEB-INF"/>
+	<property name="app.lib.dir" value="${webinf.dir}/lib"/>
+ 
+    <!--+
+        | Default classpath
+        +-->
+    <path id="classpath">
+        <fileset dir="${lib.dir}">
+        	<include name="**/*.jar" />
+        </fileset>
+    	<fileset dir="${app.lib.dir}">
+			<include name="**/*.jar" />
+		</fileset>
+        <pathelement path="${java.class.path}/" />
+    </path>
+    
+    <!--+
+        | Compiles the Java source files
+        +-->
+    <target name="compile" depends="clean">
+        <echo>Compiling classes</echo>
+        <javac destdir="${classes.dir}" classpathref="classpath" source="1.7" target="1.7" debug="on" >

```

Comment:
I think it would make sense to require Java 8 since Cascade requires Java 8, so that most users would have it and it would be easier for us to work with since we already have it and also Java 7 can no longer be downloaded [apparently since 2015](https://java.com/en/download/faq/java_7.xml).

> ### Where can I get Java 7 downloads?
> July 2015: Updates for Java 7 are no longer available to the public. Oracle offers updates to Java 7 only for customers who have purchased Java support or have Oracle products that require Java 7.

Reply:
Actually it looks like we are already using Java 8 on master.